docs/modeling/triggers.md

```diff
@@ -171,4 +172,13 @@ Actions are stored as a `payload`, for example using the send a start message ac
 		}
 	}
 }
-```
\ No newline at end of file
+```
+
+#### Execute a script
+The action to execute a script will execute a named [script](../modeling/scripts.md) that exists in the same project as the trigger.
+
+| Property | Description | Example | Required | 
+| -------- | ----------- | ------- | -------- |
+| `scriptName` | The name of the script is automatically populated from the selected script. | update-orders-script | Yes | 
+| `scriptId` | The ID of the script is automatically populated from the selected script. | 19ced673-e701-4e6c-ace6-f8aaee5455eb | Yes | 
+| `variables` | Values from the trigger can be mapped to script variables. | | No | 
```

Comment:
Just a suggestion, 'Values from the trigger `event outputs` ....'

Reply:
👍